app/views/directives/_copy-to-clipboard.html

```diff
@@ -1,11 +1,17 @@
-<div class="input-group copy-to-clipboard">
-  <input id="{{id}}" type="text" class="form-control" value="{{clipboardText}}" ng-disabled="isDisabled" ng-readonly="!isDisabled" select-on-focus>
+<div class="input-group copy-to-clipboard" style="{{displayWide ? 'max-width:100%;' : ''}}">
```

Comment:
Suggest something like

```
ng-class="{ 'limit-width': !displayWide }"
```

then update the CSS to key off that class.

https://github.com/spadgett/origin-web-console/blob/e01508561bb7d0c7f6c53dbedcf7dd42d37e0c52/app/styles/_core.less#L1208-L1210